src/M161.md

```diff
@@ -0,0 +1,38 @@
+# M161
+
+The M161 is a simple multi-cart mapper by Mani (Nintendo's official Chinese distributor). 
+This mapper only allows for 8 separate 32 KiB banks, with a limit of 1 bankswitch, preventing any bankswitches afterward.
+This gives 1 bank for the main menu, and a maximum of 7 different 32 KiB games. No SRAM can be applied to this mapper, so only purely MBC-less titles can be used.
```

Comment:
```suggestion
This gives 1 bank for the main menu, and a maximum of 7 different 32 KiB games.
No SRAM can be applied to this mapper, so only purely MBC-less titles can be used.
```

Reply:
Also maybe reword the second sentence to e.g. "This mapper does not support SRAM, so only purely MBC-less titles can be used."?